src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/VSCompletionList.cs

```diff
@@ -98,6 +107,42 @@ private static VSCompletionList PromoteCommonCommitCharactersOntoList(VSCompleti
                 Data = completionList.Data,
             };
             return promotedCompletionList;
+
+            static bool SequenceEqual(Container<string> a, Container<VSCommitCharacter> b)
```

Comment:
`Enumerable.SequenceEqual` I think does the same thing, might be able to remove this method

Reply:
The trick here is that this is `Container<string>` vs `Container<VSCommitCharacter>` and `Enumerable.SequenceEqual` only accepts sequences where the Types are the same.

Reply:
oooo ok y a that totally makes sense